SunbirdAI/noise-sensors-monitoring
# Add location category and description fields to the location model

### What does this PR do?
Adds a location category to the location model. Also adds the following extra properties based on the category field:
- **Location description**: a description of the surrounding areas e.g residential, hospital e.t.c
- **Night limit**: The DB threshold at which an area is considered noisy at night.
- **Day limit**: The DB threshold at which an area is considered noisy during the day.

The above are based on the [NEMA Noise Standards and Control Regulations](http://nema.go.ug/sites/all/themes/nema/docs/noise_standards_and_control_regulations.pdf)


### How to test this PR?
- Run the application.
- Go to the device page. Confirm that the location info is shown.
- Confirm that the category can be edited.
- Make a call to the endpoint `http://localhost:8000/devices/locations` and confirm that the fields `location_description`, `day_limit` and `night_limit` are present in the response.

Example API response:
```
[
    {
        "id": "276c67f1-5302-4ce5-996d-e152c5e6249f",
        "latitude": 0.335984,
        "longitude": 32.57291,
        "city": "Kampala",
        "division": "Central Division",
        "parish": "Wandegeya",
        "village": "N/A",
        "latest_metric": {
                .....
        },
        "location_description": "Category A: hospital, convalescence home, sanatorium, home for the aged and higher learning institute, conference rooms, public library, environmental or recreational sites",
        "day_limit": 45,
        "night_limit": 35
    }
]
```
